docs/drafts/badges.rst

```diff
@@ -0,0 +1,144 @@
+Badges
+======
+
+This is a summary of the "badge concept" user story. The concepts are
+not yet implemented in Adhocracy 3.
+
+Adhocracy 2 knows a variety of badges and similar entities. The aim is
+to streamline these entities into more consistent or possibly independent
+concepts in Adhocracy 3. Some aspects shall not be implemented in A3.
+
+This document can be replaced once the concepts are ready in A3.
+
+
+Badge features in Adhocracy 2
+-----------------------------
+
+The various badges, categories and tags can be categorized as the
+following:
+
+
+Allowed targets
+    What can be badged?
+
+
+Locality
+    Do badges exist globally or locally only?
+
+Exclusivity
+    Can only one badge out of a certain group be assigned to a target
+    object?
+
```

Comment:
Basically yes, but per "badge group". If there's a badge group "decided state" and a badge group "implemented state" (both exclusive), then the referred object can have up to one badge out of the possible "decided state" badges and up to one badge out of the possible "implemented state".